editor/src/components/PropertiesPanel/PropertiesPanelDynamic.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <v-expansion-panels
+    v-model="panel"
+    multiple
+    no-gutters
+    dense
+    class="caption"
+    :key="selectedElement.edit"
+  >
+    <v-expansion-panel>
+      <v-expansion-panel-header>
+        <div>
+          <span class="caption font-weight-bold"> General</span>
+          <v-icon small class="px-2">mdi-card</v-icon>
+        </div>
+      </v-expansion-panel-header>
+      <v-expansion-panel-content>
+        <PropertiesPanelDynamic
+          v-if="generalData"
+          :config="generalData"
+          @updateData="updateData"
+        />
+      </v-expansion-panel-content>
+    </v-expansion-panel>
+  </v-expansion-panels>
+</template>
+
+<script lang="ts">
+import PropertiesPanelDynamic from './PanelDynamic/PanelDynamic.vue';
+import { sync } from 'vuex-pathify';
+import { omit } from 'lodash';
+import { defineComponent } from '../../util/vue';
+import { getVariableName } from '../../model/uischema';
+import { tryFindByUUID } from '../../util/schemasUtil';
+import _ from 'lodash';
+const PropertiesPanel = defineComponent({
+  name: 'PropertiesPanel',
+  props: {},
+  components: {
+    PropertiesPanelDynamic,
+  },
+  data() {
+    return {
+      panel: [0, 1, 2, 3, 4],
+      generalData: null,
+      type: null,
+      uiElement: null,
+    };
+  },
+  computed: {
+    schema: sync('app/editor@schema'),
+    uischema: sync('app/editor@uiSchema'),
+    selectedElement: sync('app/editor@element'),
+  },
+  mounted() {
+    this.setSelection(this.selectedElement.selected);
```

Comment:
this.selectedElement can be undefined.

Reply:
The property is defined as computed property
 selectedElement: sync('app/editor@element'),